### PR TITLE
Rvfbrnet 36

### PR DESCRIPTION
**Pull request by AndriiKatsubo**

### CODE REVIEW

File: MedicalDocManagment.UI/src/app/views/users.list.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿<h3>Список користувачів</h3>
+
+<table>
+    <tr>   
+        <th>Ім'я користувача</th>
+        <th>Аватар</th>
+        <th>E-mail</th>
+        <th>Посада</th>
+        <th>Роль</th>
+        <th>Чи активний</th>
```
Comment (Ravennod) [inline MedicalDocManagment.UI/src/app/views/users.list.html]: IsActive field should not be displayed on this page.

File: MedicalDocManagment.UI/src/app/views/users.list.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    </tr>
+    <tr *ngFor="let user of users">
+        <td>{{user.username}}</td>
+        <td>{{user.avatar}}</td>
+        <td>{{user.email}}</td>
+        <td>{{user.position}}</td>
+        <td>{{user.role}}</td>
+        <td>{{user.isactive}}</td>
```
Comment (Ravennod) [inline MedicalDocManagment.UI/src/app/views/users.list.html]: IsActive field should not be displayed on this page.

**Review by Ravennod — DISMISSED** [review]
I have left two comments related to IsActive field.

File: MedicalDocManagment.UI/src/app/views/users.list.html (hunk 2 above)
Comment (AndriiKatsubo) [reply MedicalDocManagment.UI/src/app/views/users.list.html]: Why not?

Comment (AndriiKatsubo) [reply MedicalDocManagment.UI/src/app/views/users.list.html]: ok, nevermind

File: MedicalDocManagment.UI/src/app/users.list.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    users = this.getUsersList();
+
+    ngOnInit() {
+        this.getUsersList()
+    }
+
+    getUsersList() {
+        // mock-up
+       return [
+            { username: "Іван", email: "ivan@lala.net", position: "лікар", role: "якась роль", isactive: "так" },
```
Comment (artempanko) [inline MedicalDocManagment.UI/src/app/users.list.ts]: Please use boolean type in property isactive and follow the code convention.

**Review by artempanko — CHANGES_REQUESTED** [review]

File: MedicalDocManagment.UI/src/app/users.list.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import { Response } from '@angular/http';
+
+
+@Component({
+    selector: 'emptyContent',
+    templateUrl: './app/views/users.list.html'
+})
+
+export class UsersListComponent {
+    users = this.getUsersList();
```
Comment (artempanko) [inline MedicalDocManagment.UI/src/app/users.list.ts]: unused property users

File: MedicalDocManagment.UI/src/app/users.list.ts (hunk 3 above)
Comment (AndriiKatsubo) [reply MedicalDocManagment.UI/src/app/users.list.ts]: Where exactly do you see incompatibility with code convention?

**Comment by KostetskyiYaroslav** [comment]
It seems to be ok, but look at the new project structure. It's different. 
Also there are too many useless commits here.

File: MedicalDocManagment.UI/src/app/users.list.ts (hunk 3 above)
Comment (artempanko) [reply MedicalDocManagment.UI/src/app/users.list.ts]: > isactive

**Review by artempanko — DISMISSED** [review]
Follow the code convention.

File: MedicalDocManagment.UI/src/app/users.list.ts (hunk 3 above)
Comment (AndriiKatsubo) [reply MedicalDocManagment.UI/src/app/users.list.ts]: ok, done.

**Pull request closed without merging**